repos/system_upgrade/common/actors/targetuserspacecreator/libraries/userspacegen.py

```diff
@@ -331,6 +331,7 @@ def _get_product_certificate_path():
             ),
             reporting.Tags([reporting.Tags.REPOSITORY]),
             reporting.Flags([reporting.Flags.INHIBITOR]),
+            reportiSeverity(reporting.Severity.HIGH),
```

Comment:
```suggestion
            reporting.Severity(reporting.Severity.HIGH),
```

Reply:
@vinzenz  huh. and I yanked the line from the file using vim :D I probably did a very bad typo. Good catch!